ecosystem/sep-0010.md

```diff
@@ -199,6 +204,8 @@ Upon successful verification, **Server** responds with a session JWT, containing
 * `iat` (the time at which the JWT was issued [RFC7519, Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6)) — current timestamp (`1530644093`)
 * `exp` (the expiration time on or after which the JWT must not be accepted for processing, [RFC7519, Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4)) — a server can pick its own expiration period for the token (`1530730493`)
 * `client_domain` - (optional) a nonstandard JWT claim containing the client home domain, included if the challenge transaction contained a `client_domain` (see [Verifying Client Application Identity](#verifying-client-application-identity))
+* `memo_type` — (optional) the memo type provided in the challenge, one of `text`, `id` or `hash`. See the [Memos](#memos) section for more information.
```

Comment:
❓ Are there use cases where we need to separate authenticating for an account for all memos, vs an account with memo type none? This change doesn't currently support that. How should the memo_type being unset be interpreted?

Reply:
Now that we're using only memo IDs, there is no need for `memo_type`.